xds/internal/xdsclient/bootstrap/bootstrap.go

```diff
@@ -292,6 +261,9 @@ type Config struct {
 	// In any of those cases, it is an error if the specified authority is
 	// not present in this map.
 	Authorities map[string]*Authority
+	// NodeProto contains the Node proto to be used in xDS requests. This will be
```

Comment:
Do we need this comment `This will be of type *v3corepb.Node.`? Given that the field is defined to be of that type now.